register.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel='stylesheet' href="css/style.css"/>
+    <title>Register New</title>
+</head>
+<body background = "image/images.jpg">
+
+   <nav class = "topnav">
+         <ul>
+             <li><a href="view_products.html">Products</a></li>
+             <li><a href="sales.html">Sales</a></li>
+             <li><a href="add.html">Add</a></li>
+             <li><a href="edit.html">Edit</a></li>
+             <li><a href="index.html">Logout</a></li>
+         </ul>
+     </nav>
+
+        <main class="boxed">   
+       
+            <h1><i>REGISTER</i></h1>
+
+           
+            <label for="first name">
+               <b> First Name</b>
+            </label><br>
+            <input type="text" placeholder="First Name">
+            <br>
+            
+            <label for="last name">
+               <b>Last Name</b>
+            </label><br>
+            <input type="text" placeholder="Last Name">
+            <br>
+            
+            <label for="username">
+                <b>Username</b>
+            </label><br>
+            <input type="text" placeholder="Username">
+            <br>
+            
+            <label for="contact">
+               <b> Contact</b>
+            </label><br>
+            <input type="tel" placeholder="Telephone Number">
+            <br>
+            
+            <label for="email">
+               <b> Email</b>
+            </label><br>
+            <input type="email" placeholder="Email">
+            <br>
+            
+            <label for="password">
+                  <b> Create Password</b> 
+            </label><br>
+            <input type="password" placeholder="Create Password">
+            <br>
+            
+            <label for="password">
```

Comment:
If a label has a for attribute, There should be an element on the page that has an id similar to the for attribute